frontend/amundsen_application/static/js/ducks/lineage/api/v0.ts

```diff
@@ -14,7 +14,7 @@ export function getTableLineage(
   const tableQueryParams = getQueryParams({ key, depth, direction });
 
   return axios
-    .get(`${API_PATH}/get_table_lineage?${tableQueryParams}`)
+    .get(`${API_PATH}/get_table_lineage?${tableQueryParams}`, { timeout: 90000 })
```

Comment:
Let's move this number into a constant

Reply:
my bad this was a test I will clean it up